docs/index.md

```diff
@@ -290,24 +290,24 @@ title: Whiting index
 
 Notes on the image captions
 ---------------------------
-* **`?`**
+### N1
 
   The scanned image doesn't match the description by Whiting.
 
-* **&iquest;?**
+### N2
 
   The patch is too small and shows only edge cases. It needs at least three repeats to be unambiguous.
 
-* **&mdash;**
+### N3
 
   The ground uses sewings, three-pair stitches, odd number of threads, tallies and/or picots.
   None of these are supported by GroundForge.
 
-* **#**
+### N4
 
   Bandages: dull for the objective of this page and too elaborate.
   
-* **`x`** short links
+### N5 short links
```

Comment:
```suggestion
### N5
```
Also start the first phrase with something like: "This note is about the links after `[N5]:`, they" or use your own words to explain these links.